packages/features/shell/Shell.tsx

```diff
@@ -73,12 +73,31 @@ import {
   ChevronDown,
   Copy,
   Settings,
-  Slack,
   Users,
   Zap,
   User as UserIcon,
 } from "@calcom/ui/components/icon";
 
+const discordIcon = () => {
+  return (
+    <div className="flex-shrink-0">
+        <svg
+          className="h-4 w-4"
+          fill="currentColor"
+          xmlns="http://www.w3.org/2000/svg"
+          width="24px"
+          height="24px"
+          viewBox="0 0 24 24"
+        >
+          <title>
+            Discord Logo
+          </title>
+          <path d="M18.9419 5.52317C17.6473 4.91744 16.263 4.47723 14.8157 4.22656C14.638 4.5479 14.4304 4.98012 14.2872 5.32394C12.7487 5.09258 11.2245 5.09258 9.7143 5.32394C9.57116 4.98012 9.3588 4.5479 9.17947 4.22656C7.73067 4.47723 6.3448 4.91906 5.05016 5.52638C2.43887 9.47238 1.73099 13.3204 2.08493 17.1137C3.81688 18.4071 5.49534 19.1928 7.14548 19.7069C7.55291 19.1462 7.91628 18.5501 8.22931 17.9219C7.63313 17.6954 7.06211 17.4158 6.52256 17.0912C6.6657 16.9852 6.80571 16.8743 6.94098 16.7603C10.2318 18.2995 13.8074 18.2995 17.0589 16.7603C17.1958 16.8743 17.3358 16.9852 17.4774 17.0912C16.9362 17.4174 16.3637 17.6969 15.7675 17.9235C16.0805 18.5501 16.4423 19.1478 16.8513 19.7085C18.503 19.1944 20.183 18.4087 21.915 17.1137C22.3303 12.7163 21.2056 8.90361 18.9419 5.52317ZM8.67765 14.7809C7.68977 14.7809 6.87963 13.8586 6.87963 12.7355C6.87963 11.6125 7.67247 10.6886 8.67765 10.6886C9.68285 10.6886 10.493 11.6109 10.4757 12.7355C10.4772 13.8586 9.68285 14.7809 8.67765 14.7809ZM15.3223 14.7809C14.3344 14.7809 13.5243 13.8586 13.5243 12.7355C13.5243 11.6125 14.3171 10.6886 15.3223 10.6886C16.3275 10.6886 17.1376 11.6109 17.1203 12.7355C17.1203 13.8586 16.3275 14.7809 15.3223 14.7809Z"/>
+        </svg>
+    </div>
+  )
+}
+
```

Comment:
i wonder if we should make this an export, and reuse on our 404 page, too: http://cal.com/404-page